### PR TITLE
LPS-177534 Hide in the contents panel the content display restricted contents

### PR DESCRIPTION
**Pull request by sandrodw3**
cc @lfbesada @p2kmgcl (special guest @victorg1991)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by sandrodw3** [comment]
ci:test:sf

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/FragmentGeneralPanel.js [hunk 1]
```diff
@@ -35,6 +37,8 @@ import {FieldSet} from './FieldSet';
 export function FragmentGeneralPanel({item}) {
 	const dispatch = useDispatch();
 
+	const restrictedItemIds = new Set(config.restrictedItemIds);
```
[the remark this answers is not in the document]
Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/FragmentGeneralPanel.js]: Ups, my fault 😅 updated, thank you!

**Comment by ealonso** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:forward

**Review by p2kmgcl — APPROVED** [review]

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**